lib/cosi/cosi.go

```diff
@@ -191,15 +191,25 @@ func (c *Cosi) CreateResponse() (*Response, error) {
 
 // Response generates the response from the commitment, challenge and the
 // responses of its children.
-func (c *Cosi) Response(responses []*Response) (*Response, error) {
-	// create your own response
-	if err := c.genResponse(); err != nil {
-		return nil, err
+// if respond == true, will also add our response
+func (c *Cosi) Response(respond bool, responses []*Response) (*Response, error) {
```

Comment:
As just discussed now: I would really like to have this integrated to be able to port BFTCoSi over to the new library.